code/modules/admin/verbs/randomverbs.dm

```diff
@@ -472,6 +472,30 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	admin_ticket_log(M, msg)
 	SSblackbox.add_details("admin_verb","Rejuvinate") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/cmd_admin_freeze(mob/living/M in GLOB.mob_list)
+	set category = "Special Verbs"
+	set name = "Freeze Player"
+	if(!holder)
+		to_chat(src, "Only administrators may use this command.")
+		return
+	if(!istype(M))
+		return
+
+	if(!M.adminfrozen)
+		var/sleepies = M.AmountSleeping()
+		M.adminfrozen = sleepies ? sleepies : 1
+		M.SetSleeping(20000)//2k seconds to get your admin shit together

```

Comment:
`INFINITY` instead of `20000`?

Reply:
Not really necessary. 2k will be more than enough and things might get unexpectedly funky.

Reply:
2k is 33 mins.